source/_integrations/sharkiq.markdown

```diff
@@ -17,6 +17,8 @@ ha_integration_type: integration
 
 The `sharkiq` integration allows you to control your [Shark IQ](https://www.sharkclean.com/vacuums/robot-vacuums/) vacuum.
 
+*Please note that this is currently broken. There are work arounds which can be found [here](https://github.com/home-assistant/core/issues/123392)*
```

Comment:
We do not document issues, those should be tracked in the issue tracker (as linked).

If we add all issues to every documentation page, it would become unmaintainable.

Nevertheless, thank you for being willing to contribute 👍 

../Frenck